tpm/caps.go

```diff
@@ -0,0 +1,76 @@
+package tpm
+
+import (
+	"context"
+	"fmt"
+	"slices"
+
+	"github.com/google/go-tpm/legacy/tpm2"
+	"go.step.sm/crypto/tpm/algorithm"
+)
+
+type Capabilities struct {
+	Algorithms []algorithm.Algorithm
+}
+
+func (c *Capabilities) SupportsAlgorithms(algs ...algorithm.Algorithm) bool {
```

Comment:
Did you want to change it to singular, or did you change your mind? Don't forget the doc on the struct and the method.

Reply:
[d378c7d](https://github.com/smallstep/crypto/pull/580/commits/d378c7d363e5a5c9691dcae6e3184863058fc287)